cmd/deploy.go

```diff
@@ -626,8 +681,8 @@ func (c deployConfig) Validate(cmd *cobra.Command) (err error) {
 
 	// Check Image Digest was included
 	// (will be set on the function during .Configure)
-	var digest string
-	if digest, err = imageDigest(c.Image); err != nil {
+	var digest bool
+	if digest, err = hasDigest(c.Image); err != nil {
```

Comment:
minor nit:  we called this `digested` above, so probably worth updating to match. It's easy to mistake this for the digest value itself (which is what it was originally named for)